install.md

```diff
@@ -0,0 +1,96 @@
+# Install
+
+This is a basic install: Juggluco is installed on your phone, connected to your sensor and your smartwatch will display the values when connected to the phone.
+
+</br>
+
+## Prerequisite
+
+Your smartwatch must be paired with the smartphone you will use to setup GlucoDataHandler.
+
+</br>
+
+## 1 - Install GlucoDataHandler on your phone
+
+On your phone go the the [Releases](https://github.com/pachi81/GlucoDataHandler/releases) page and select the last (top) pre-release or release.  
+Download and install `GlucoDataHandler.apk` (allow install from unknown source if required).
+
+<img src='images/download.png' width=350 align=left>
+
+</br>
+
+Open the app, it will look like this:
+
+<img src='images/installed.png' width=400 align=left>
+
+</br>
+
+Open Juggluco and enable `Glucodata broadcast` in Settings. Save and Ok.
+
+<img src='images/broadcast.png' width=700 align=left>
+
+</br>
+
+You should now also see your BG display in the GlucoDataHandler app.
+
+<img src='images/broadcast_ok.png' width=300 align=left>
+
+</br>
+
+## 2 - Install GlucoDataHandler on your watch
+
+Now download `GlucoDataHandler-Wear.apk` on your phone but **do not install it**.
+
+<img src='images/download.png' width=350 align=left>
+
+### Method #1 - Install with Wear Installer 2
```

Comment:
Just a question, does it is working for you?
I have tried Wear Installer 1 and 2, but both apps are showing always success, even if the installation fails. Except of one time, it haven't installed anything...

Reply:
No, it failed with both when I tried... but a few days ago I help a guy in the UK and he managed to install with Wear Installer... Same pre-release... I used ADB that's the most reliable way I know.